src/app/space/create/deployments/apps/deployment-card.component.html

```diff
@@ -27,11 +27,24 @@
     </div>
     <div [collapse]="collapsed">
       <deployments-donut [applicationId]="applicationId" [mini]="false"></deployments-donut>
-      <p>CPU Usage</p>
-      <pfng-chart-sparkline [config]="cpuConfig" [chartData]="cpuData"></pfng-chart-sparkline>
-      <br>
-      <p>Memory Usage</p>
-      <pfng-chart-sparkline [config]="memConfig" [chartData]="memData"></pfng-chart-sparkline>
+      <div class="deployment-chart">
+        <div class="chart-column">
+          <pfng-chart-sparkline class="chart-sparkline" [config]="cpuConfig" [chartData]="cpuData"></pfng-chart-sparkline>
+        </div>
+        <div class="chart-column chart-text">
+          <p class="chart-info-header">CPU (Cores)</p>
+          <p>{{ cpuVal }} of 10</p>
+        </div>
+      </div>
```

Comment:
Think you could extract this out into a component? It's already copy-pasted twice here for two charts, and the design mock shows three (plus another different line chart).